src/index.ts

```diff
@@ -79,6 +79,3 @@ export * from 'semantic-ui-react'
 
 // Colors
 export * from './colors'
-
-// Types
-export * from './types/avatar'
```

Comment:
Should we export the `Avatar` type from `@dcl/schemas` with this UI lib to make it easier for to use the type without the need to explicitly import `@dcl/schemas`?